src/IFrameRoutes.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import {Route} from 'react-router-dom';
+import {ButtonHero} from './ButtonCatalog';
+import {CardHero} from './CardCatalog';
+import {CheckboxHero} from './CheckboxCatalog';
+import {ChipsHero} from './ChipsCatalog';
+import {DialogHero} from './DialogCatalog';
+import {DrawerHero} from './DrawerCatalog';
+import {ElevationHero} from './ElevationCatalog';
+import {FabHero} from './FabCatalog';
+import {IconButtonHero} from './IconButtonCatalog';
+import {ImageListHero} from './ImageListCatalog';
+import {LayoutGridHero} from './LayoutGridCatalog';
+import {ListHero} from './ListCatalog';
+import {MenuHero} from './MenuCatalog';
+import {RadioButtonHero} from './RadioButtonCatalog';
+import {RippleHero} from './RippleCatalog';
+import {SelectHero} from './SelectCatalog';
+import {SliderHero} from './SliderCatalog';
+import {SnackbarHero} from './SnackbarCatalog';
+import {SwitchHero} from './SwitchCatalog';
+import {TabsHero} from './TabsCatalog';
+import {TextFieldHero} from './TextFieldCatalog';
+import {TopAppBarHero} from './TopAppBarCatalog';
+import {TypographyHero} from './TypographyCatalog';
+import {LinearProgressHero} from './LinearProgressIndicatorCatalog';
+import { matchPath } from 'react-router';
```

Comment:
was this package already installed? If not, we should try to use the `props.match`